selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/NewWorkspace.java

```diff
@@ -200,17 +705,146 @@ public void typeToFiltersInput(String value) {
     seleniumWebDriverHelper.setValue(filterStackInput, value);
   }
 
+  public void deleteLastTagFromInputTagsField() {
+    seleniumWebDriverHelper.waitAndSendKeysTo(filterStackInput, BACK_SPACE.toString());
+    seleniumWebDriverHelper.sendKeys(BACK_SPACE.toString());
```

Comment:
Looks like duplication of `seleniumWebDriverHelper.waitAndSendKeysTo(filterStackInput, BACK_SPACE.toString())`

Reply:
In this case, we should send one "backspace" exactly to defined tag (for activating) and next "backspace" should be a simple button pressing. This is for tests side. From users side, it looks like twice "backspace" button pressing.

Reply:
I see. In that case I would propose to change signature of `SeleniumWebDriverHelper#waitAndSendKeysTo()` so that it accepts `CharSequence...`. This allows us to send several keys at a time.